learn/wavemaker-release-notes/v10-14-0.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "WaveMaker 10.14.0 - Release date: 1 May 2022"
+id: ""
+sidebar_label: "Release v10.14.0"
+---
+*WaveMaker released another version update with new features and bug fixes.*
+
+## New Features
+---
+
+### SSL for Deployed App URLs
+
+Introducing SSL for Deployed Apps to [Demo Cloud](/learn/app-development/deployment/pipelines-phases#deployment-environment), allowing you to access apps with the secure protocol HTTPS. To support this, we made changes to the App URL pattern. Below are the changes.
+
+**WMO Personal workspace**
+
+Old: `http://p8888888n45.cloud.wavemakeronline.com/TestApp`  
+New: `http://cloud.wavemakeronline.com/pk8888888n45/TestApp`  
+
+:::note
+In the URL, `pk8888888n45` is a Project key.
+:::
+
+**Teams and Enterprise**
+
+Old: `http://wmshowcase.cloud.wavemakeronline.com/TestApp`  
+New: `http://cloud.wavemakeronline.com/wmshowcase/TestApp`  
+
+:::note
+`wmshowcase` is the WaveMaker Online team's name and WaveMaker Enterprise's name in the URL.
+:::
+
+For more information, see [Demo Cloud](/learn/app-development/deployment/pipelines-phases#deployment-environment).
+
+## Improvements
+---
+
+- Docker-compose accept env variables with character `_`.
```

Comment:
Who did this ?
Docker-compose accept env variables with character ??

Reply:
@udaysa